### PR TITLE
[kube-prometheus-stack] prom operator update / add recording and alerting switch to kubeScheduler

### PR DESCRIPTION
**Pull request by lusson-luo**
Signed-off-by: lusson <lusson@foxmail.com>

#### What this PR does / why we need it

- updates all components (prom operator 0.60.1 etc)
- can disabled kubeScheduler alert or record alone, before kubeSchedulerAlert and kubeSchedulerRecord control by one switch kubeScheduler

#### Which issue this PR fixes

- according to prs #2308 #2397 

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by lusson-luo — COMMENTED** [review]
finish my review

**Review by monotek — CHANGES_REQUESTED** [review]
This would require new major version, as an existing config value is changed

**Review by monotek — CHANGES_REQUESTED** [review]
See also: https://github.com/prometheus-community/helm-charts/pull/2410/files#L3

**Comment by lusson-luo** [comment]
> First-time contributors need a maintainer to

ok,i upgrade major version to 40.0.0

**Comment by lusson-luo** [comment]
> See also: https://github.com/prometheus-community/helm-charts/pull/2410/files#L3

@monotek ok,i upgrade major version to 40.0.0

**Review by monotek — CHANGES_REQUESTED** [review]
Plaese raise version to 41.0.0

Readme needs upgrading instructions.

I would like to do this together with nex prometheus operator update.

**Comment by lusson-luo** [comment]
> Plaese raise version to 41.0.0
> 
> Readme needs upgrading instructions.
> 
> I would like to do this together with next prometheus operator update.

@monotek ok, thanks
1. i raise version to 41.0.0
2. The original readme no related configuration, so i think don't upgrading Readme.
3. Okay, I'm waiting for other prometheus operator update.

**Comment by monotek** [comment]
> @monotek ok, thanks
> 

> 2. The original readme no related configuration, so i think don't upgrading Readme.

All major versions need an entry in the updating section, describing whats changed and what to do to get old behaviour.

**Comment by lusson-luo** [comment]
> > @monotek ok, thanks
> 
> > 2. The original readme no related configuration, so i think don't upgrading Readme.
> 
> All major versions need an entry in the updating section, describing whats changed and what to do to get old behaviour.

understand, i add major versions updating section in readme

File: charts/kube-prometheus-stack/README.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 40.x to 41.x
```
Comment (monotek) [inline charts/kube-prometheus-stack/README.md:83]:
```suggestion
### From 40.x to 41.x

```

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/README.md (hunk 1 above)
Comment (lusson-luo) [reply charts/kube-prometheus-stack/README.md:83]: i add space line

**Comment by lusson-luo** [comment]
@monotek readme has add empty line

**Review by monotek — APPROVED** [review]

**Pull request merged**